services/catarse/app/actions/shared/addresses/countries.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Shared
+  module Addresses
+    class Countries < Actor
+      output :response
```

Comment:
**Problema**
Faltou o tipo do output aqui, e seria melhor chamar isso de countries pois é isso que ele retorna e não um response, senão quando for usar essa action, teria que usar: `result.response` não deixando claro o que é esse response.

```suggestion
      output :countries, type: Array
```